backend/routes/auth.js

```diff
@@ -3,6 +3,9 @@ const router = express.Router()
 const mongoose = require("mongoose")
 const User = mongoose.model("User")
 const bcrypt = require("bcryptjs")
+const jwt = require("jsonwebtoken")
+const {JWT_SECRET}= require("../keys")
```

Comment:
Keys wordt niet geused dus je moet zo weer met een environnement variable doen net zoals MONGOURL

Reply:
Een optie hiervoor zou kunnen zijn:
JWT_SECRET = process.env.JWT ? process.env.JWT : require('./keys').JWT_SECRET;

Vergeet niet de JWT toe te voegen dan aan de config vars in heroku